src/System.Net.Sockets/ref/4.1.0/System.Net.Sockets.depproj

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I thought we didn't need .depproj files anymore.

@ericstj You mentioned that we would do "package harvesting" to find the previous RTM version of a contract?